src/foam/dashboard/view/DAOTable.js

```diff
@@ -171,12 +180,17 @@ foam.CLASS({
       name: 'viewMoreAction',
       label: 'View more activities',
       code: function() {
-        this.stack.push(this.StackBlock.create({
-          view: {
-            class: this.DAOBrowseControllerView,
-            data: this.dao.where(this.predicate),
-          }, parent: this.__subContext__
-        }));
+        if ( ! (this.viewMoreMenuItem === undefined ||
```

Comment:
Is this the same as just: if ( this.viewMoreMenuItem ) ?

Reply:
yeah, tested out in the js console, will change this.